src/pointfree/bichain.js

```diff
@@ -0,0 +1,21 @@
+/** @license ISC License (c) copyright 2018 original and current authors */
```

Comment:
Should be 2019

Reply:
Also gonna wanna add this file to `pointfree/index` and throw some specs in `pointfree/index` and in the main `index`